projects/xregistry/README.md

```diff
@@ -0,0 +1,78 @@
+
+## XRegistry Artwork
+
+This directory contains artwork in standard formats for XRegistry. We've prepared artwork in 2 formats (PNG & SVG), 3 layouts:
+
+- horizontal (also known as landscape format),
+- stacked (which is closer to square), and
```

Comment:
"Which has the simplified logo over of the project name"

might be a better description of the stacked format

Reply:
Good suggested, updated. Thanks!